common/containers/container.go

```diff
@@ -105,25 +105,31 @@ func (c *Container) Name() string {
 //     a.R.s
 //     R.s
 //
-// If aliases or abbreviations are configured for the container, then alias names appear after
-// the potential set of container-based names.
+// If aliases or abbreviations are configured for the container, then alias names will take
+// precedence over containerized names.
```

Comment:
Consider "...then alias expansion is done instead of searching the container path."